### ISSUE TITLE
Accessibility Issues Identified with LiaScript

### ISSUE THREAD

**Issue by tbaker-mc** [LiaScript/LiaScript#248]
We had one of our internal accessibility specialists review LiaScript for any potential accessibility issues and found the following:
* Tables -- tables are being identified as "grids." When the headers are read by a screen reader (both JAWS and NVDA were used), the table's header and the "sort" feature are read together.
* Links, text in color HEX#4E9092 -- this fails for contrast at a 3.7:1 ratio
* Focus indicator (in both light and dark mode) -- fails for contrast 
* Focus indicator -- no focus indicator is visible on "next page" navigation

**Comment by mgifford** [comment]
Do you have any additional information on how to replicate this @tbaker-mc ?

**Comment by mgifford** [comment]
Thanks. But where do you see this? How do I replicate it? Xpaths? HTML snippets? I am very familiar with web accessibility but need to know where you are finding these problems.  It isn't in the Markdown for instance

**Comment by tbaker-mc** [comment]
Hi @mgifford -- we aren't quite sure if we understand your question. 

For testing accessibility, we tested a SCORM file in our LMS and in the LiaScript web viewer. This is the specific course we used: https://liascript.github.io/course/?https://raw.githubusercontent.com/usds/ditap-curriculum-update/refs/heads/main/3_Curriculum/3A_DITAP-Core-Curriculum/Module-1/Sprint-3/Self-Paced%20Materials/1_Digital%20Service%20Tech%20Bootcamp.md#1
